packages/mdc-snackbar/foundation.ts

```diff
@@ -173,62 +145,44 @@ class MDCSnackbarFoundation extends MDCFoundation {
     }
   }
 
-  /**
-   * @return {boolean}
-   */
-  getCloseOnEscape() {
+  getCloseOnEscape(): boolean {
     return this.closeOnEscape_;
   }
 
-  /**
-   * @param {boolean} closeOnEscape
-   */
-  setCloseOnEscape(closeOnEscape) {
+  setCloseOnEscape(closeOnEscape: boolean) {
     this.closeOnEscape_ = closeOnEscape;
   }
 
-  /**
-   * @param {!KeyboardEvent} evt
-   */
-  handleKeyDown(evt) {
-    if (this.getCloseOnEscape() && (evt.key === 'Escape' || evt.keyCode === 27)) {
+  handleKeyDown(evt: KeyboardEvent) {
+    const isEscapeKey = evt.key === 'Escape' || evt.keyCode === 27;
+    if (isEscapeKey && this.getCloseOnEscape()) {
       this.close(REASON_DISMISS);
     }
   }
 
-  /**
-   * @param {!MouseEvent} evt
-   */
-  handleActionButtonClick(evt) {
+  handleActionButtonClick(_evt: MouseEvent) {
```

Comment:
can you remove this argument since you're not using it?

Reply:
I considered doing removing it, but I thought it would be more future-proof to keep it.

If we end up needing to inspect the event object in the future—e.g., to avoid taking an action if the shift/meta keys are pressed—we won't need to introduce a breaking change, because we'll already have access to the event.

Removing the param would technically be a breaking change too, so we'd have to create a separate PR to make the change in `master` as well.

Reply:
Yeah... from the perspective of this being a straight conversion, might as well leave it, since I guess it was unused even on master then, right?